state/utils/ttl.go

```diff
@@ -38,3 +38,19 @@ func ParseTTL(requestMetadata map[string]string) (*int, error) {
 	}
 	return nil, nil
 }
+
+// ParseTTL64 parses the "ttlInSeconds" metadata property.
+func ParseTTL64(requestMetadata map[string]string) (*int64, error) {
+	val, found := requestMetadata[MetadataTTLKey]
+	if found && val != "" {
```

Comment:
Nit: the `found` check here is redundant. If the value doesn’t exist, `Val` will be an empty string already.

Reply:
Agreed. This was a copy+paste from the `int` implementation in the same package. I'll update both.